crates/runtime/src/dataaccelerator.rs

```diff
@@ -260,6 +263,9 @@ pub async fn create_accelerator_table(
     acceleration_settings: &acceleration::Acceleration,
     secrets: Arc<RwLock<Secrets>>,
 ) -> Result<Arc<dyn TableProvider>> {
+    make_spice_data_directory()
+        .map_err(|err| Error::AccelerationCreationFailed { source: err.into() })?;
```

Comment:
I don't think we want to do this unless we know we need to store something there. I can imagine situations of Spice is running in production where creating random folders could cause issues because of permissions issues.